v_3_2_5/master_template.go

```diff
@@ -1827,13 +1827,13 @@ write_files:
         {{ range .Hyperkube.Apiserver.Pod.CommandExtraArgs -}}
         - {{ . }}
         {{ end -}}
-        - --allow_privileged=true
-        - --insecure_bind_address=0.0.0.0
+        - --allow-privileged=true
+        - --insecure-bind-address=0.0.0.0
         - --anonymous-auth=false
         - --insecure-port=0
-        - --kubelet_https=true
+        - --kubelet-https=true
         - --kubelet-preferred-address-types=InternalIP
-        - --secure_port={{.Cluster.Kubernetes.API.SecurePort}}
+        - --secure-port={{.Cluster.Kubernetes.API.SecurePort}}
```

Comment:
Those are deprecated we should replace them with some sort of config.

Reply:
right kubelet is now dynamically configurable (you can even hotswap configuration IIRC) but it needs to be a config file I think